linux_os/guide/system/permissions/restrictions/sysctl_kernel_modules_disabled/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: fedora,ol8,rhel7,rhel8
+
+title: 'Disable loading and unloading of kernel modules'
+
+description: '{{{ describe_sysctl_option_value(sysctl="kernel.modules_disabled", value="1") }}}'
+
+rationale: |-
+    Malicious kernel modules can have a significant impact on system security and
+    availability. Disabling loading of kernel modules prevents this threat. Note
+    that once this option has been set, it cannot be reverted without doing a
+    system reboot. Make sure that all needed kernel modules are loaded before
+    setting this option.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: CCE-83392-1
+    cce@rhel8: CCE-83397-0
```

Comment:
These CCEs were not subtracted from the pool

Reply:
They are here: https://github.com/ComplianceAsCode/content/pull/6534/files#diff-cfce205679ce81560e4155abb87229ae85690b3d7299297a24032ce39232fc29L1

:D

Reply:
Thank you, missed this.